src/Raven.Server/Documents/AttachmentsStorage.cs

```diff
@@ -1359,5 +1390,10 @@ public static IEnumerable<BlittableJsonReaderObject> GetAttachmentsFromDocumentM
                 }
             }
         }
+
+        [IndexEntryKeyGenerator]
```

Comment:
Can we rename `IndexEntryKeyGenerator` to `StorageIndexEntryKeyGenerator` - to make it explicit it's about index storage not ravendb indexes

Reply:
done